main.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+// 各種DOMを取得
+const btn = document.querySelector('#btn');
+const textbox = document.querySelector('#textbox');
+const tbody = document.querySelector('#tbody');
+// タスクを格納する配列を定義
+const todos = [];
+
+// カウンターを定義
+let counter = 0;
+
+// todoを追加する関数
+function createTask() {
+  const todo = {
+    id: counter,
+    task: textbox.value,
+    status: '作業中'
+  };
+  todos.push(todo);
+  counter++;
+  textbox.value = '';
+};
+
+// 状態ボタンを生成する関数
+function createStatusBtn(currentTodo) {
+  const statusBtn = document.createElement('td');
+  const btn = document.createElement('button');
+  btn.innerText = currentTodo.status;
+  statusBtn.appendChild(btn);
+  return statusBtn;
+};
+
+// タスクを削除する関数
+function deleteTask(event) {
+  const arrayId = event.path[2].innerText[0];
+  todos.splice(arrayId, 1);
+  tbody.innerHTML = '';
+  for(let i = 0; i < todos.length; i++){
+   todos[i].id = i;
+  }
+  todos.forEach(function(todo){
```

Comment:
匿名関数よりもアロー関数の方がイマドキですし、これから使われる記法と思います。

```suggestion
  todos.forEach((todo) => {
```